russian/call_friend/call_friend.md

```diff
@@ -0,0 +1,5 @@
+ Видя стену из зефира вы звоните своему корешу с вопросом чем он вас таким вчера угощал. Вам рекомендуют выпить чегото бодрящего а потом прогулятся и все пройдет. И вы не замедляя 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.